### PR TITLE
Change master and nodes type of aws cluster

### PR DESCRIPTION
**Pull request by chandankumar4**
Signed-off-by: Chandan Kumar <chandan.kr404@gmail.com>

**What this PR does / why we need it**:

- Change master and nodes instance type from `t2.large` to `t2.xlarge`

**Special notes for your reviewer**:

image

### CODE REVIEW

File: k8s/aws/k8s-installer/vars.yml [hunk 1]
```diff
@@ -5,7 +5,7 @@ cidr_block: 10.0.0.0/16
 cidr_block2: 10.0.1.0/24
 destination_cidr: 0.0.0.0/0
 image: ami-6b3fd60c
-master_size: t2.large
-node_size: t2.large
+master_size: t2.xlarge
```
Comment (ksatchit) [inline k8s/aws/k8s-installer/vars.yml]: Master instance type may still be lesser than the nodes, I guess!

Comment (chandankumar4) [reply k8s/aws/k8s-installer/vars.yml]: okay! i'll assign the master to `t2.large`

**Comment by ksatchit** [comment]
Will take this based on results of recent CI runs. We will revisit the instance types based on usage stats.

**Comment by chandankumar4** [comment]
Okay!

**Pull request merged**